codice/ddf
# DDF-3888 updated docs templates for standalone integrator docs

#### What does this PR do?

updates documentation templates to create standalone integrator documentation artifact.

#### Who is reviewing it? 

@mcalcote @ahoffer @austinsteffes @garrettfreibott 

#### Select relevant component teams: 

@codice/docs 

#### Committers to review/merge.

@clockard
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?

Confirm that builds are successful and `integrating.html` is available. (run with `-Prelease` to create `integrating.pdf`)

#### What are the relevant tickets?
[DDF-3888](https://codice.atlassian.net/browse/DDF-3888)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
